### PR TITLE
wet-boew release 4.0.34

### PR DESCRIPTION
**Pull request by namjohn920**
Release for 4.0.34

### CODE REVIEW

File: site/pages/docs/versions/dwnld-en.hbs [hunk 1]
```diff
@@ -41,13 +41,13 @@
 		<tbody>
 			<tr>
 				<td>Web Experience Toolkit (WET)</td>
-				<td>v4.0.33 (STR) <a href="https://wet-boew.github.io/wet-boew-documentation/decision/9.html" class="label label-info">What is STR?</a></td>
-				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/wet-boew/releases/latest">Download<span class="wb-inv"> - v4.0.33 - Web Experience Toolkit (WET) theme</span></a></td>
+				<td>v4.0.34 (STR) <a href="https://wet-boew.github.io/wet-boew-documentation/decision/9.html" class="label label-info">What is STR?</a></td>
```
Comment (duboisp) [inline site/pages/docs/versions/dwnld-en.hbs]: Remove (STR) this is going to be a normal release

File: site/pages/docs/versions/dwnld-fr.hbs [hunk 2]
```diff
@@ -46,13 +46,13 @@
 		<tbody>
 			<tr>
 				<td>Boîte à outils de l'expérience Web (WET-BOEW)</td>
-				<td>v4.0.33 (DCT) <a href="https://wet-boew.github.io/wet-boew-documentation/decision/9.html" class="label label-info">Qu'est-ce que le DCT/<span lang="en">STR</span>?</a></td>
-				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/wet-boew/releases/latest">Télécharger<span class="wb-inv"> - v4.0.33 - Thème de la Boîte à outils de l'expérience Web (WET-BOEW)</span></a></td>
+				<td>v4.0.34 (DCT) <a href="https://wet-boew.github.io/wet-boew-documentation/decision/9.html" class="label label-info">Qu'est-ce que le DCT/<span lang="en">STR</span>?</a></td>
```
Comment (duboisp) [inline site/pages/docs/versions/dwnld-fr.hbs]: Remove (DCT)

File: site/pages/docs/versions/dwnld-en.hbs [hunk 3]
```diff
@@ -80,8 +102,8 @@
 				<tr>
 					<td>Canada.ca theme (GCWeb)</td>
 					<td>v4.0.32</td>
-					<td><a href="https://github.com/wet-boew/GCWeb/archive/v6.0.zip">Download<span class="wb-inv"> - v6.0 source code - Canada.ca theme</span> (theme only)</a></td>
-					<td><a href="https://github.com/wet-boew/GCWeb/releases/download/v6.0/themes-dist-6.0-gcweb.zip">Download<span class="wb-inv"> - v6.0 production files - Canada.ca theme</span></a></td>
+					<td><a href="https://github.com/wet-boew/GCWeb/archive/v6.0.zip">Download<span class="wb-inv"> - v6.1 source code - Canada.ca theme</span> (theme only)</a></td>
```
Comment (GormFrank) [inline site/pages/docs/versions/dwnld-en.hbs]:
```suggestion
					<td><a href="https://github.com/wet-boew/GCWeb/archive/v6.1.zip">Download<span class="wb-inv"> - v6.1 source code - Canada.ca theme</span> (theme only)</a></td>
```

File: site/pages/docs/versions/v4.0/v4.0.34-en.hbs [hunk 4]
```diff
@@ -0,0 +1,144 @@
+---
+{
+	"title": "v4.0.34 (STR) release notes",
+	"language": "en",
+	"description": "v4.0.34 (STR) release notes - Version history - Web Experience Toolkit (WET)",
+	"altLangPrefix": "v4.0.34",
+	"dateModified": "2020-03-17"
+}
+---
+<p>Version 4.0.34 (STR) <a href="https://wet-boew.github.io/wet-boew-documentation/decision/9.html" class="label label-info">What is STR?</a></p>
```
Comment (duboisp) [inline site/pages/docs/versions/v4.0/v4.0.34-en.hbs]: REmove (STR) and include the release of 4.0.33 or a link to it please

File: site/pages/docs/versions/dwnld-en.hbs [hunk 5]
```diff
@@ -80,8 +102,8 @@
...
+					<td><a href="https://github.com/wet-boew/GCWeb/releases/download/v6.0/themes-dist-6.0-gcweb.zip">Download<span class="wb-inv"> - v6.1 production files - Canada.ca theme</span></a></td>
```
Comment (GormFrank) [inline site/pages/docs/versions/dwnld-en.hbs]:
```suggestion
					<td><a href="https://github.com/wet-boew/GCWeb/releases/download/v6.1/themes-dist-6.1-gcweb.zip">Download<span class="wb-inv"> - v6.1 production files - Canada.ca theme</span></a></td>
```

File: site/pages/docs/versions/v4.0/v4.0.34-fr.hbs [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+			<li>Firefox 70.0.1</li>
+			<li>Firefox 69.0.3</li>
+			<li>Firefox ESR - 68.2.0</li>
+			<li>IE 11</li>
+			<li>Edge 44</li>
+		</ul>
+	</section>
+</section>
+<section>
+	<h2>Détails</h2>
```
Comment (duboisp) [inline site/pages/docs/versions/v4.0/v4.0.34-fr.hbs]: The following don't match the english list

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Also update the list of browser supported

**Comment by GormFrank** [comment]
Also please change the Release data as tomorrow

File: site/pages/docs/versions/dwnld-en.hbs (hunk 3 above)
Comment (namjohn920) [reply site/pages/docs/versions/dwnld-en.hbs]: It is under 6.0. Does this still have to be 6.1?

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
Looks good,
@GormFrank after your review, you can merge. Then we will need to wait the github build to be completed before to create the new release tag.

**Review by GormFrank — APPROVED** [review]

**Pull request merged**